_layouts/contact_us.html

```diff
@@ -15,11 +15,17 @@
   
   {{ page.intro_content | markdownify }} 
   
+  {% if site.contact_phone_number_enabled and site.contact_phone_number and 
+     site.contact_phone_number_location == 'top' %} 
+    {% include contact_phone.html common_applications=layout.common_applications %} 
```

Comment:
I see it existed previously, but I'm not sure if `common_applications` is necessary for this partial? It seems to be relevant only for `contact_form.html`.

Reply:
Good catch. Thank you.